pyproject.toml

```diff
@@ -33,7 +33,7 @@ classifiers = [
 ]
 requires-python = ">= 3.11, < 4"
 dependencies = [
-  "frequenz-api-common >= 0.6.5, < 0.7",
+  "frequenz-api-common @ git+https://github.com/frequenz-floss/frequenz-api-common@v0.x.x",
```

Comment:
I guess this sneaked into this commit accidentally?

Reply:
removed